lib/src/main/kotlin/com/swmansion/starknet/data/types/transactions/Transaction.kt

```diff
@@ -41,11 +42,35 @@ sealed class Transaction {
     abstract val type: TransactionType
 }
 
+@OptIn(ExperimentalSerializationApi::class)
+@Serializable
+data class TransactionHashPair(
+    @Serializable(with = TransactionPolymorphicSerializer::class)
+    @SerialName("transaction")
+    @JsonNames("txn")
+    val transaction: Transaction,
+
+    @SerialName("transaction_hash")
+    @JsonNames("txn_hash")
+    val hash: Felt,
+)
```

Comment:
Not sure this is a way to go. I'd rather we didn't merge those two into one data class - perhaps there could be some other data class that is super class to them?

Reply:
And I would definitely change the name if we leave it as it is

Reply:
This was removed in 0.4-rc4 and replaced with allO, will go back to using Transaction class.